gapic/src/main/com/google/gapid/models/CommandStream.java

```diff
@@ -233,19 +234,6 @@ protected void onUiThread(Path.CommandTreeNode result) {
     CommandStream.Node root = this.getData();
     ListenableFuture<TreePath> result = getTreePath(root, Lists.newArrayList(root),
         index.getNode().getIndicesList().iterator());
-    if (index.isGroup()) {
-      // Find the deepest group/node in the path that is not the last child of its parent.
-      result = MoreFutures.transform(result, path -> {
-        while (path.getSegmentCount() > 0) {
-          CommandStream.Node node = (CommandStream.Node)path.getLastSegment();
-          if (!node.isLastChild()) {
-            break;
-          }
-          path = path.getParentPath();
-        }
-        return path;
-      });
-    }
```

Comment:
Trying to delete it since 1. This part of code is not being used: before this PR there's no longer `CommandIndex` initialized with the `group` value set to be true. 2. `Find the deepest group/node in the path that is not the last child of its parent.` doesn't seem to handle the aim of finding the group, especially this will do wrong to a corner case: e.g. the path passed in is already a valid group but happens to be the last child, then the path be wrongly transformed into one of its parents(seen this bug in testing).

The `group` variable will be turned into use to config how gapis resolve a command index into a real tree node.

Reply:
A corner case: 
--Submit 0
----FrameBuffer0                      <- (c)
------Renderpass0
------Renderpass1
------Renderpass2                    <- (b)
--------vkBeginRenderPass.    <- (a)
----FrameBuffer1

Although our intention towards group handling here is to jump back from (a) to (b), but we would actually jump back to (c), since it happens to be the last renderpass.